Add Russian version

Dmitriy-Vas opened the pull request.

Dmitriy-Vas committed:
Add Russian translation.

Add the Russian translation in a separate file (as standard) and link in
English translation file (cross-reference)

Dmitriy-Vas committed:
Grammar and spelling corrections.

Remove abandoned english text

Dmitriy-Vas commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+* Это генерирует "чистую" историю, без ненужных слияющих коммитов
+* _Что видишь - то и получаешь_, т.е. при обзоре кода все изменения идут от определенного коммита, избегая скрытые изменения в слияющих коммитах 
+* Больше слияний решаются автором коммита, и каждое слитое изменение находится в коммите с соответствующим сообщением
```
Need help with translation this line:
* More merges are resolved by the committer, and every merge change is in a commit with a proper message

lex111 replied:
```suggestion
* Больше слияний решаются автором коммита, поэтому каждое слитое изменение находится в коммите с соответствующим сообщением
```

Dmitriy-Vas commented on README_ru-RU.md:
```diff
@@ -256,37 +256,31 @@ See also: #456, #789
     * Это необычно - просматривать слияющие коммиты, поэтому избегая их, вы гарантируете что все изменения имеют коммит, к которому они относятся 
 
-### When to squash
 ### Когда же "Squash" коммиты?
```
Need a good alternative to the word "Squash" in russian

RomuloOliveira reviewed (dismissed):
@Dmitriy-Vas thank you for your PR, really appreciate it!

Since I'm not able to do a thorough review (an automatic translator is far from ideal to do this job), do you know some colleagues that could help? No problem if you don't, just let me know and I'll merge it, I was lucky that people helped me and reviewed others languages. Thanks again for your contribution!

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+# Руководство по работе с коммитами
```
```suggestion
# Руководство по написанию коммитов
```

Dmitriy-Vas replied:
This guide explains not only "writing" commits, so i think "working" will be better.

lex111 replied:
Если по _работе_, то предполагается команды или инструкции, но здесь по **большей части** именно написание.

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+[![Скажи Спасибо!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+Руководство для понимания важности коммитов и как правильно их писать. 
```
```suggestion
Руководство для понимания важности сообщений коммитов и того, как правильно их писать. 
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+## Что такое "коммит"?
+
+Простыми словами, коммит это _снимок_ ваших локальных файлов, записанный в ваш локальный репозиторий.
```
```suggestion
Простыми словами, коммит - это _снимок_ ваших локальных файлов, записанный в локальный репозиторий.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+## Что такое "коммит"?
+
+Простыми словами, коммит это _снимок_ ваших локальных файлов, записанный в ваш локальный репозиторий.
+Вопреки тому, что думают некоторые, [git хранит не только изменения между файлами, он хранит полную версию всех файлов](https://git-scm.com/book/eo/v1/Ekkomenci-Git-Basics#Snapshots,-Not-Differences).
```
```suggestion
Вопреки тому, что думают некоторые, [git хранит не только изменения между файлами, но и в целом полную версию всех файлов](https://git-scm.com/book/ru/v2/%D0%92%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5-%D0%9E%D1%81%D0%BD%D0%BE%D0%B2%D1%8B-Git).
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+- Чтобы ускорить и упростить проверку кода
+- Чтобы помочь в понимании изменений
+- Чтобы объяснить вопросы, которые не могут быть описаны только кодом
+- Чтобы помочь будущим разработчикам разобраться почему и как были сделаны изменения, облегчив поиск и устранение ошибок
```
```suggestion
- Чтобы помочь будущим разработчикам разобраться, почему и как были сделаны изменения, облегчив поиск и устранение ошибок
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+## Полезные практики
+
+Это некоторые практики, собранные из моего опыта, статей и других руководств. Если у вас есть предложение (или вы не согласны с некоторым) не стесняйтесь открыть Pull Request и внести свой вклад.
```
```suggestion
Это некоторые практики, собранные из моего опыта, статей и других руководств. Если у вас есть предложение (или вы не согласны с некоторым) не стесняйтесь открыть пулреквест и внести свой вклад.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+_Но зачем использовать императивную (повелительную) форму?_
+
+Сообщения коммита описывает что описанные изменения **делают**, их эффект, а не что было сделано.
```
```suggestion
Сообщение коммита описывает, что конкретно **делают** зафиксированные изменения, его конечный результат, а не что было сделано.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+Сообщения коммита описывает что описанные изменения **делают**, их эффект, а не что было сделано.
+
+[Эта замечательная статья от Chris Beams](https://chris.beams.io/posts/git-commit/) даёт нам простое описание того, что может быть использовано, чтобы лучше писать сообщения коммитов в императивной форме:
```
```suggestion
В [замечательной статье Криса Бимса (Chris Beams)](https://chris.beams.io/posts/git-commit/) ([перевод на русский](https://habr.com/ru/post/416887/)) приводится шаблон предложения, который может использоваться, чтобы лучше написать сообщения коммитов в императивной форме:
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,447 @@
+If applied, this commit will used InventoryBackendPool to retrieve inventory backend
+```
+
+### Используйте заглавную первую букву
````
```suggestion
### Начинайте сообщение коммита с заглавной буквы
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,447 @@
+add `use` method to Credit model
+```
+
+Причина, по которой первая буква должна быть заглавной, - в следовании грамматическому правилу об использовании заглавных букв в начале предложения.
````
```suggestion
Смысл этой рекомендации простой: в соответствии с правилами грамматики первое слово предложения начинается с заглавной буквы.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+Причина, по которой первая буква должна быть заглавной, - в следовании грамматическому правилу об использовании заглавных букв в начале предложения.
+
+Использование этой практики может варьироваться от человека к человеку, от команды к команде или даже от языка к языку.
+Заглавная или нет, важно придерживаться единому стандарту и следовать ему.
```
```suggestion
Заглавная или нет, важно придерживаться единого стандарта и следовать ему.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+Использование этой практики может варьироваться от человека к человеку, от команды к команде или даже от языка к языку.
+Заглавная или нет, важно придерживаться единому стандарту и следовать ему.
+
+### Попытайтесь сообщить изменения без надобности смотреть исходный код
```
```suggestion
### Старайтесь описывать изменения так, чтобы указывать на исходный код
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,447 @@
+Adjust css
+```
+
+Это полезно во многих случаях (пр. несколько коммитов, различные изменения и рефакторинг), для помощи проверяющим понять о чём думал автор коммита.
````
```suggestion
Это полезно во многих случаях (например, когда несколько коммитов, различные изменения и рефакторинг), чтобы помочь проверяющим понять, что хотел предложить автор коммита.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+Это полезно во многих случаях (пр. несколько коммитов, различные изменения и рефакторинг), для помощи проверяющим понять о чём думал автор коммита.
+
+### Используйте описание в сообщении для объяснения "почему", "для чего", "как" и дополнительные детали 
```
```suggestion
### Используйте тело сообщения для объяснения "почему", "для чего", "как" и дополнительные детали 
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,447 @@
+setup a new attribute (in_use_amount) with a new value
+```
+
+Тема и описание сообщения разделены пустой линией.
````
```suggestion
Тема и описание сообщения разделены пустой строкой.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+Тема и описание сообщения разделены пустой линией.
+
+Дополнительные пустые линии считаются частью описания.
```
```suggestion
Дополнительные пустые строки считаются частью описания.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+Дополнительные пустые линии считаются частью описания.
+
+Такие символы как `-`, `*` и \ улучшают общую читаемость.
```
```suggestion
Такие символы как `-`, `*` и ` улучшают общую читаемость.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+### Сохраняйте согласованность языка
+
+Для владельцев проектов: Выберите язык и пишите все сообщения коммитов используя этот язык. В идеале, он должен совпадать с комментариями в коде, стандартной локалью (для переведенных проектов), и т.д.
```
```suggestion
Для владельцев проектов: Выберите язык и пишите все сообщения коммитов на этом языке. В идеале, он должен совпадать с комментариями в коде, локалью по умолчанию (для проектов с локализацией), и т.д.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+Для владельцев проектов: Выберите язык и пишите все сообщения коммитов используя этот язык. В идеале, он должен совпадать с комментариями в коде, стандартной локалью (для переведенных проектов), и т.д.
+
+Для помощников: Пишите ваши сообщения коммитов используя тот же язык, что и в истории коммитов.
```
```suggestion
Для участников проекта: Пишите сообщения коммитов на том языке, который используется в истории коммитов.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+### Шаблон
+
+Этот шаблон, [который изначально описан Tim Pope](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), также появляется в [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project).
```
```suggestion
Этот шаблон, [который изначально описан Тимом Поупом (Tim Pope)](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), также упоминается в [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project).
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+## Перебазировка vs. Слияние
+
+Эта секция является кратким пересказом превосходного руководства Atlassian, ["Merging vs. Rebasing"](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
```
```suggestion
Этот раздел является кратким пересказом превосходного руководства ["Merging vs. Rebasing"](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) от Atlassian.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+![](https://wac-cdn.atlassian.com/dam/jcr:01b0b04e-64f3-4659-af21-c4d86bc7cb0b/01.svg?cdnVersion=hq)
+
+### Перебазировка
```
```suggestion
### Перебазирование
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+### Перебазировка
+
+**Кратко:** Применяет ваше ответвление коммитов, один за другим, к базовой ветви, генерируя новое дерево. 
```
```suggestion
**Кратко:** Применяет ваши коммиты в ветке, один за другим, к базовой ветки (например, master), генерируя новое дерево. 
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+### Слияние
+
+**Кратко:** Создаёт новый коммит, называемый (соответственно) _слияющим коммитом_ (_merge commit_), с различиями между двумя ветвями.
```
```suggestion
**Кратко:** Создаёт новый коммит, называемый (соответственно) _коммит слияния_ (_merge commit_), с различиями между двумя ветками.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+### Почему некоторые люди предпочитают перебазирование вместо слияния?
+
+Я, в частности, предпочитаю перебазирование вместо слияния. Причины включают в себя: 
```
```suggestion
Я, например, предпочитаю перебазирование вместо слияния по следующим причинам:
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+Я, в частности, предпочитаю перебазирование вместо слияния. Причины включают в себя: 
+
+* Это генерирует "чистую" историю, без ненужных слияющих коммитов
```
```suggestion
* Это создает "чистую" историю, без ненужных коммитов-слияний
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+Я, в частности, предпочитаю перебазирование вместо слияния. Причины включают в себя: 
+
+* Это генерирует "чистую" историю, без ненужных слияющих коммитов
+* _Что видишь - то и получаешь_, т.е. при обзоре кода все изменения идут от определенного коммита, избегая скрытые изменения в слияющих коммитах 
```
```suggestion
* _Что видишь, то и имеешь,_, т.е. при ревью кода все изменения идут от определенного коммита, без скрытых изменений в коммитах при слиянии
```

Dmitriy-Vas committed:
Use common russian words instead of borrowed from english

Remove abandoned english sentences
Add russian alternative to "Squash": "сжать"

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+* Это генерирует "чистую" историю, без ненужных слияющих коммитов
+* _Что видишь - то и получаешь_, т.е. при обзоре кода все изменения идут от определенного коммита, избегая скрытые изменения в слияющих коммитах 
+* Больше слияний решаются автором коммита, и каждое слитое изменение находится в коммите с соответствующим сообщением
+    * Это необычно - просматривать слияющие коммиты, поэтому избегая их, вы гарантируете что все изменения имеют коммит, к которому они относятся 
```
```suggestion
    * Обычно не смотрят коммиты-слияния, так что таким образом все изменения зафиксированы в коммите, к которому они непосредственно относятся 
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+* Больше слияний решаются автором коммита, и каждое слитое изменение находится в коммите с соответствующим сообщением
+    * Это необычно - просматривать слияющие коммиты, поэтому избегая их, вы гарантируете что все изменения имеют коммит, к которому они относятся 
+
+### Когда же "Squash" коммиты?
```
```suggestion
### Когда нужно делать слияние коммитов?
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+### Когда же "Squash" коммиты?
+
+"Squashing" это процесс объединения серии коммитов в один коммит. 
```
```suggestion
Слияние (Squashing) — это процесс объединения множества коммитов в один-единственный коммит. 
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+"Squashing" это процесс объединения серии коммитов в один коммит. 
+
+It's useful in several situations, e.g.:
```
```suggestion
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+"Squashing" это процесс объединения серии коммитов в один коммит. 
+
+It's useful in several situations, e.g.:
+Это полезно в нескольких ситуациях, пример:
```
```suggestion
Это может пригодиться в ряде ситуаций, например:
```

Dmitriy-Vas replied:
Error:
```
Это может пригодиться в ряде ситуациях, например:
```
Fix:
```
Это может пригодиться в ряде ситуаций, например:
```

lex111 replied:
Исправил

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+It's useful in several situations, e.g.:
+Это полезно в нескольких ситуациях, пример:
+
+- Уменьшение количества коммитов с небольшим контекстом или без контекста (исправление ошибок, форматирование, забытые вещи)
```
```suggestion
- Уменьшение количества коммитов с небольшим контекстом или без контекста (исправление опечаток, форматирование, дополнения)
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+Это полезно в нескольких ситуациях, пример:
+
+- Уменьшение количества коммитов с небольшим контекстом или без контекста (исправление ошибок, форматирование, забытые вещи)
+- Объединение отдельных изменений, которые имеют больше смысла, когда применяются вместе
```
```suggestion
- Объединение независимых изменений, которые лучше всего применять вместе
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+
+- Уменьшение количества коммитов с небольшим контекстом или без контекста (исправление ошибок, форматирование, забытые вещи)
+- Объединение отдельных изменений, которые имеют больше смысла, когда применяются вместе
+- Переписывание _WIP_ (работа в прогрессе) коммитов
```
```suggestion
- Переписывание коммитов в стадии _WIP_ (черновика)
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,447 @@
+- Объединение отдельных изменений, которые имеют больше смысла, когда применяются вместе
+- Переписывание _WIP_ (работа в прогрессе) коммитов
+
+### When to avoid rebase or squash?
```
```suggestion

```

Dmitriy-Vas commented:
@RomuloOliveira thank you too for your work!

I'm new user in GitHub and still don't have colleagues who can help me with translations (and reviews).
I just reviewed previous commit and correct some words.
Please merge it when you are done with the review.

lex111 reviewed (commented):
Don't rush it, don't rush it!

@Dmitriy-Vas Thank you for this :wink: 


UPD: I will continue the review later. For now, current suggestions can be accepted.

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,445 @@
+Adjust css
+```
+
+Это полезно во многих случаях (пр. несколько коммитов, различные изменения и рефакторинг), чтобы помочь проверяющим понять о чём думал автор коммита.
````
```suggestion
Это полезно во многих случаях (пр. несколько коммитов, различные изменения и рефакторинг), чтобы помочь проверяющим понять о чём думал автор коммита.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+Сообщения коммита описывает что описанные изменения **делают**, их эффект, а не что было сделано.
+
+[Эта замечательная статья от Chris Beams](https://chris.beams.io/posts/git-commit/) даёт нам простое описание того, что может быть использовано, чтобы лучше писать сообщения коммитов в повелительном наклонении:
```
```suggestion
В [замечательной статье Криса Бимса (Chris Beams)](https://chris.beams.io/posts/git-commit/) ([перевод на русский](https://habr.com/ru/post/416887/)) приводится шаблон предложения, который может использоваться, чтобы лучше писать сообщения коммитов в императивной форме:
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+* Больше слияний решаются автором коммита, и каждое слитое изменение находится в коммите с соответствующим сообщением
+    * Это необычно - просматривать слияющие коммиты, поэтому избегая их, вы гарантируете что все изменения имеют коммит, к которому они относятся 
+
+### Когда же "сжимать" коммиты?
```
```suggestion
### Когда нужно делать слияние коммитов?
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+### Когда же "сжимать" коммиты?
+
+Сжатие - это процесс объединения серии коммитов в один коммит. 
```
```suggestion
Слияние (Squashing) — это процесс объединения множества коммитов в один-единственный коммит.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+### Когда избегать перебазирования или сжатия?
+
+Избегайте этого в публичных коммитах или в общих ветвях, где работают несколько человек.
```
```suggestion
Избегайте этого в публичных коммитах или в общих ветках, в которых работают несколько человек.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+### Когда избегать перебазирования или сжатия?
+
+Избегайте этого в публичных коммитах или в общих ветвях, где работают несколько человек.
+Это также перезаписывает историю коммитов и существующие коммиты, делая это с коммитами которые находятся в разных ветвях (пр., коммиты отправленные в удаленный репозиторий или которые идут из других веток) могут вызвать путаницу и люди могут потерять их изменения (как локальные, так и удаленные) из-за разрозненного дерева и конфликтов.
```
```suggestion
Перебазирование и слияние также перезаписывает историю коммитов и существующие коммиты, делая это с коммитами, которые находятся в разных ветвках (например, коммиты, отправленные в удаленный репозиторий или коммиты из других веток) могут вызвать путаницу и поэтому можно потерять изменения (как локальные, так и удаленные) из-за различающегося дерева и конфликтов.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+### rebase -i
+
+Используйте их чтобы сжать коммиты, изменить сообщения, переписать/удалить/реорганизовать коммиты, и т.д.
```
```suggestion
Используйте их, чтобы объединить коммиты, изменить сообщения, переписать/удалить/изменить порядок коммитов и т.д.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+#### fixup
+
+Используйте для очистки коммитов без необходимости в перебазировке. 
```
```suggestion
Используйте эту команду, чтобы привести в порядок коммиты, не прибегая к перебазированию. 
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+#### fixup
+
+Используйте для очистки коммитов без необходимости в перебазировке. 
+[Данная статья](http://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html) имеет хорошие примеры того, как и когда это делать.
```
```suggestion
В [этой статье](http://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html) есть хорошие примеры того, как и когда это делать.

```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+### cherry-pick
+
+Это очень полезно для применения коммитов, которые вы сделали в неправильной ветви, без необходимости писать их заново.
```
```suggestion
Эта команда может быть очень полезной для применения коммита, который был сделан в неверной ветке, чтобы не писать код заново.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+### add/checkout/reset [--patch | -p]
+
+Давайте представим что мы имеем следующий список различий:
```
```suggestion
Давайте представим, что у нас есть следующий список различий:
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
++
++This is a template, [written originally by Tim Pope](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), which appears in the [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project).
++
+ ## Contributing
```
```suggestion
 ## Участие в проекте
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,445 @@
++- [A Note About Git Commit Messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+```
+
+Мы можем использовать `git add -p` чтобы добавить только те патчи, которые нам нужны, без нужды изменять уже написанный код.
````
```suggestion
Мы можем выполнить `git add -p` чтобы добавить только те патчи, которые нам нужны, без необходимости изменять уже существующий код.
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,445 @@
+Split into 2 hunks.
+```
+
+#### hunk 1
````
```suggestion
#### часть 1
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,445 @@
+Stage this hunk [y,n,q,a,d,/,j,J,g,e,?]?
+```
+
+#### hunk 2
````
```suggestion
#### часть 2
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,445 @@
+
+```
+
+#### hunk 3
````
```suggestion
#### часть 3
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+https://whatthecommit.com/
+
+## Нравится руководство?
```
```suggestion
## Нравится это руководство?
```

lex111 commented on README_ru-RU.md:
````diff
@@ -0,0 +1,445 @@
++- [A Note About Git Commit Messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+```
+
+## Другие интересные вещи
````
```suggestion
## Кое-что интересное
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+## Помощь
+
+Любой вид помощи будет оценен. Пример тем, с которыми вы можете помочь:
```
```suggestion
Любая помощь приветствуется. Вот, например, что вы можете сделать:
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+
+Любой вид помощи будет оценен. Пример тем, с которыми вы можете помочь:
+
+- Ошибки в грамматике и правописании 
```
```suggestion
- Исправить ошибки в грамматике и правописании 
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,445 @@
+- Ошибки в грамматике и правописании 
+- Перевод на другие языки
+- Улучшение списка связанных источников
+- Неверная или неполная информация
```
```suggestion
- Изменить неверную или неполную информацию
```

lex111 reviewed (changes requested):
@RomuloOliveira I did a review with the suggestions, after accepted them, this PR can be merged!

@Dmitriy-Vas вы можете применить мои предложения одним коммитом, посмотрите эту статью https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change

lex111 committed:
Update Russian translation

Common good-explained Russian sentences
Links to translated articles

Co-Authored-By: Dmitriy-Vas <Dmitriy.Vas.B@gmail.com>

lex111 committed:
Apply suggestions from code review

Co-Authored-By: Dmitriy-Vas <Dmitriy.Vas.B@gmail.com>

Dmitriy-Vas commented:
> 
> 
> @RomuloOliveira I did a review with the suggestions, after accepted them, this PR can be merged!
> 
> @Dmitriy-Vas вы можете применить мои предложения одним коммитом, посмотрите эту статью https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change

Mhm, didn't know this works from "files" tab, added 1 excess commit.

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,446 @@
+
+### Когда нужно делать слияние коммитов?
+
+Слияние (Squashing) — это процесс объединения множества коммитов в один-единственный коммит.
```
```suggestion
Объединение (Squashing) — это процесс объединения множества коммитов в один-единственный коммит.
```

lex111 commented on README_ru-RU.md:
```diff
@@ -0,0 +1,446 @@
+* Больше слияний решаются автором коммита, поэтому каждое слитое изменение находится в коммите с соответствующим сообщением
+    * Обычно не смотрят коммиты-слияния, так что таким образом все изменения зафиксированы в коммите, к которому они непосредственно относятся 
+
+### Когда нужно делать слияние коммитов?
```
```suggestion
### Когда нужно делать объединение коммитов?
```

lex111 reviewed (changes requested):
Final corrections

lex111 committed:
Apply suggestions from code review

Co-Authored-By: Dmitriy-Vas <Dmitriy.Vas.B@gmail.com>

lex111 reviewed (approved):
LGTM :+1:

RomuloOliveira reviewed (approved):
Thank you very much @Dmitriy-Vas and @lex111
